tests/unit/integrations/admin/indexing-notification-integration-test.php

```diff
@@ -198,13 +202,17 @@ public function test_register_hooks_cleanup_notification() {
 			->once();
 
 		Monkey\Functions\expect( 'wp_next_scheduled' )
-			->once()
+			->twice()
 			->andReturn( true );
 
 		Monkey\Actions\expectAdded( Indexing_Notification_Integration::NOTIFICATION_ID )
 			->with( [ $this->instance, 'create_notification' ] )
 			->once();
 
+		$this->options_helper
+			->expects( 'get' )
+			->once();
+
```

Comment:
I would set an expectation on which option is retrieved as well as mock the value that is returned.
E.g.:
```php
		$this->options_helper
			->expects( 'get' )
			->with( 'indexing_reason' )
			->once()
			->andReturn( '' );
```